src/input/parse_json.rs

```diff
@@ -1,52 +1,11 @@
 use std::fmt;
 
 use pyo3::prelude::*;
-use pyo3::types::{PyDict, PyList, PySet};
+use pyo3::types::{PyDict, PyList};
 use serde::de::{Deserialize, DeserializeSeed, Error as SerdeError, MapAccess, SeqAccess, Visitor};
 
 use crate::lazy_index_map::LazyIndexMap;
 
-use crate::build_tools::py_err;
-
-#[derive(Copy, Clone, Debug)]
-pub enum JsonType {
-    Null = 0b10000000,
-    Bool = 0b01000000,
-    Int = 0b00100000,
-    Float = 0b00010000,
-    String = 0b00001000,
-    Array = 0b00000100,
-    Object = 0b00000010,
-}
```

Comment:
Although we won't need this anymore, I'm really curious why we had C-style tags here. What did that do for us that plain enums didn't? I was kinda confused when I saw `json_mask: u8`.